packages/binding-rust/webview-sys/build.rs

```diff
@@ -0,0 +1,56 @@
+extern crate cc;
+extern crate pkg_config;
+
+use std::{
+    env,
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+fn main() {
+    let webview_path = PathBuf::from("../../webview");
```

Comment:
I think its still problematic in the long term to ref folders outside of the package this way...